devel/install-local-tools.sh

```diff
@@ -214,20 +237,20 @@ function install-go-tools
     }
 
     # Install godoc
-    verbose go get -u -v golang.org/x/tools/cmd/godoc || die "Installing godoc"
+    (cd && GO11MODULE=on verbose go get -u -v golang.org/x/tools/cmd/godoc) || die "Installing godoc"
 
     # Install debugger
-    verbose go get -u -v github.com/go-delve/delve/cmd/dlv || die "Installing dlv"
+    (cd && GO11MODULE=on verbose go get -u -v github.com/go-delve/delve/cmd/dlv) || die "Installing dlv"
 
     # Install linter
-    verbose go get -u -v golang.org/x/lint/golint || die "Installing golint"
+    (cd && GO11MODULE=on verbose go get -u -v golang.org/x/lint/golint) || die "Installing golint"
 
     # Install kustomize
     # install-kustomize-from-source || die "Installing kustomize"
-    GO111MODULE=on verbose go get sigs.k8s.io/kustomize/kustomize/v3@v3.2.3 || die "Installing kustomize"
+    (cd && GO111MODULE=on verbose go get -v sigs.k8s.io/kustomize/kustomize/v3@v3.2.3) || die "Installing kustomize"
```

Comment:
INFO Installing on this way, the binary is installed into our GOPATH directory and doesn't modify our go.mod file into our project.

More information here: https://maelvls.dev/go111module-everywhere/